magento/pwa-studio
# Switching positions of npm and yarn in the scaffold script for CI.

## Description

Switching the positions of npm and yarn in the scaffold script for the [new CI job](https://github.com/magento-commerce/pwa-studio-cicd/pull/79).

Should merge https://github.com/magento-commerce/pwa-studio-cicd/pull/79 along with this. Also should change the branch on the code build job to default (develop) before merging for the changes to take action.

## Related Issue

Closes PWA-1480

## Acceptance

Should have a new code build job that will run scaffolding using `create-pwa`.

### Verification Stakeholders

@sharkySharks 

## Verification Steps

1. Run the [code build job](https://console.aws.amazon.com/codesuite/codebuild/913454440186/projects/pwa-scaffold-create-pwa/history?region=us-east-1&builds-meta=eyJmIjp7InRleHQiOiIifSwicyI6e30sIm4iOjIwLCJpIjowfQ).
2. Should pass without any errors. Usually takes around 9-10 mins.

## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
